sql/firefox_desktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  COUNT(*) AS mau,
+  SUM(CAST(last_seen_date = submission_date AS INT64)) AS dau,
```

Comment:
```suggestion
  COUNTIF(last_seen_date = submission_date) AS dau,
```

I believe this is logically equivalent.